docs/settingUp.md

```diff
@@ -56,6 +68,8 @@ Eclipse IDE is our preferred development environment. Support requests related t
    * [Google Plugin for Eclipse](https://developers.google.com/eclipse/docs/download). Use the correct version for your Eclipse IDE.<br>
      Be careful to omit other plugins shown on the screen (e.g Google App Engine Tools for Android, GWT plugin).
    * The latest stable [TestNG Eclipse plugin](http://testng.org/doc/download.html).
+   
+   **Verification** :  You can check if the plugins were installed successfully by going to `Help → Eclipse Installation Details`.
```

Comment:
@wkurniawan07 oh my mistake.Its just **Installation Details**.Thank you for pointing it out.